### ISSUE TITLE
OpenVINO inference on ONNX model fail  (yolov5 6.0)

### ISSUE THREAD

**Issue by viki9492** [ultralytics/yolov5#5195]
## ❔Question
I try to deploy the exported onnx model with openvino , but failed in ReadNetwork(). Is there any problem in exporting method or else?

**Comment by glenn-jocher** [comment]
@viki9492 I don't know about openVINO specifically, but you can test your ONNX model with detect.py before trying to use it elsewhere:
```bash
python export.py --weights yolov5s.pt --include onnx
python detect.py --weights yolov5s.onnx
```

**Comment by viki9492** [comment]
> @viki9492 I don't know about openVINO specifically, but you can test your ONNX model with detect.py before trying to use it elsewhere:
> 
> ```shell
> python export.py --weights yolov5s.pt --include onnx
> python detect.py --weights yolov5s.onnx
> ```

Thanks! I have found the problem.  Setting --opset lower is ok now.

**Comment by glenn-jocher** [comment]
@viki9492 good news 😃! Your original issue may now be fixed ✅ in PR #6179. This PR brings native OpenVINO export and inference:

```python
!python export.py --weights yolov5s.pt --include openvino  # export
!python detect.py --weights yolov5s_openvino_model/yolov5s.xml  # inference
!python val.py --weights yolov5s_openvino_model/yolov5s.xml --data ...  # validation
```

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
